modules/executive/api/fetchExecutiveVoteTally.ts

```diff
@@ -32,36 +40,48 @@ export async function fetchExecutiveVoteTally(chief: Chief): Promise<any | null>
     }
   });
 
-  const withDeposits = await Promise.all(
-    voters.map(voter =>
-      getChiefDeposits(voter, chief).then(deposits => ({
-        address: voter,
-        deposits
-      }))
-    )
+  // Split voters array into chunks of 1000 addresses
+  const chunkSize = 1000;
```

Comment:
Could also be helpful for future maintainers to leave a short comment about it being set to 1000 due the batch request limits